lib/identity_config.rb

```diff
@@ -176,6 +176,11 @@ def self.store
     config.add(:idv_contact_phone_number, type: :string)
     config.add(:idv_max_attempts, type: :integer)
     config.add(:idv_min_age_years, type: :integer)
+    config.add(
+      :idv_resolution_default_vendor,
+      type: :symbol,
+      enum: [:instant_verify, :mock],
+    )
```

Comment:
This, it turns out, is a much better thing to do than raise errors in application code if the setting is misconfigured